docs-source/spring/content/development/_index.md

```diff
@@ -0,0 +1,38 @@
+This section provides information about how to develop and deploy Spring Boot applications
+with the Oracle Backend for Spring Boot and Microservices. 
+
+Spring Boot applications can be developed the normal way, with no special requirements, and
```

Comment:
Change to "Spring Boot applications can be developed with no special requirements and can be deployed ..."